tutorials/stac/python/python-read-stac/index.ipynb

```diff
@@ -164,7 +186,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
```

Comment:
I will setup a hook (#5) to remove notebook metadata so these changes don't appear in future PRs.